card/lib/card/format.rb

```diff
@@ -224,7 +224,11 @@ def hide_views args
     end
     
     def parse_view_visibility val
-      (val || '').split( /[\s\,]+/ ).map { |view| canonicalize_view view }
+      if val.kind_of? Array
+        val.map { |view| canonicalize_view view }
+      else
+        (val || '').split( /[\s\,]+/ ).map { |view| canonicalize_view view }
```

Comment:
This will squeeze out empty elements of the list.  Is that was is wanted?  (i.e.  a,b,,c won't have an empty string between b and c, so you can't get that if that use case is wanted)